packages/components/src/Label/_docs/Label.stickersheet.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Label } from "../index"
+
+export default {
+  title: "KAIO-staging/Label",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Header headings={["Default", "Disabled", "Prominent"]} />
+      <StickerSheet.Body>
+        <StickerSheet.Row rowTitle="Base">
+          <Label labelText="Label Text" reversed={isReversed}></Label>
+          <Label labelText="Label Text" reversed={isReversed} disabled></Label>
+          <Label
+            labelText="Label Text"
+            reversed={isReversed}
+            variant="prominent"
+          ></Label>
+        </StickerSheet.Row>
+      </StickerSheet.Body>
+    </StickerSheet>
+  ),
+}
+
+export const StickerSheetDefault: StickerSheetStory = {
+  ...StickerSheetTemplate,
+  name: "Sticker Sheet (Default)",
+}
+
+export const StickerSheetReversed: StickerSheetStory = {
+  ...StickerSheetTemplate,
+  name: "Sticker Sheet (Reversed)",
+  parameters: { backgrounds: { default: "Purple 700" } },
+  args: { isReversed: true },
+}
+
+export const StickerSheetRTL: StickerSheetStory = {
+  ...StickerSheetTemplate,
+  name: "Sticker Sheet (RTL)",
+  parameters: { textDirection: "rtl" },
+}
```

Comment:
To make use of this, I think you need to add a few more examples using `children` and `labelText` in combination (which, if I look at it, seems like the `children` should be the main text and `labelText` should be subtext?)